src/core/domain-services/crux-messenger.ts

```diff
@@ -249,7 +252,9 @@ export class SecureContext {
             data,
         };
         const serializedSecurePacket = JSON.stringify(securePacket);
-
+        if (typeof recipientId === "string") {
```

Comment:
Receiving string recipientId  should never be the case! `recipientId: CruxId` is the definition of the method

Reply:
testing it and finding the root cause, I had put this as a temp fix.

Reply:
found and fixed